packages/components/src/Calendar/CalendarPopover/_docs/CalendarPopover.stickersheet.stories.tsx

```diff
@@ -42,14 +44,11 @@ const CalendarPopoverExample = ({
       />
       <CalendarPopover
         referenceElement={referenceElement}
-        popperOptions={{
-          modifiers: [
-            {
-              name: "offset",
-              options: {
-                offset: [24, 0],
-              },
-            },
+        floatingOptions={{
```

Comment:
I think instead of making a separate Responsive one, we should have one base only. If changing the middleware makes a big difference, then it should be reflected. Ideally we don't do strange things in our stories which might not match actual implementation.

Reply:
Fair point, I'll have to tweak a couple of things to add it to the top one, it'll still need to be absolute otherwise it won't test well

Reply:
Interdesting, looks like the snapshots are being take before the calendar fills our the available space 🤔

Reply:
So more height means less height... makes sense
![Screenshot 2024-05-22 at 4 51 43 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/70d14cdf-797b-42f0-a421-5eb563a36770)

Reply:
So the set viewport resolved this issue (Hurray!) I suspect that this is all due to the way Chromatic snaps a page with fixed or sticky elements. Seemingly no amount of additional time or space added to the bottom of the document if it thinks it has to scroll 🤷 Probably a good simulation of real world behaviour (which we test in the responsive sticker sheet) but not what we want in this particular story